### DISCUSSION TITLE
CP2K GPU Meeting

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#2612] [Team Posts]
Dear @cp2k/cp2k-developers,
you might remember that during the last CP2K developers meeting we had decided to have a meeting dedicated to CP2K and GPUs because the discussion would have been too special for the larger crowd of people that attended the CP2K developers meeting.
Thus, to find a good time slot for this CP2K-GPU meeting (2 hours) I would be happy if everyone who is invested or interested in this discussion could fill out the survey at
https://terminplaner.dfn.de/dDlBZkgrYurzxyaU
till the 15th of June.
Also, if you already have an aspect or topic in mind to talk about, please feel free to add it to the agenda and/or slides at
https://docs.google.com/presentation/d/1PbWzatrLbVOtc6m6UCBD97w8HNKJYWfQ_JeCga8Cod8/edit?usp=sharing
Many Thanks

**Comment by robertschade** [comment]
Dear @cp2k/cp2k-developers,
we would like to invite you to the CP2K GPU Developers meeting on Juli 7th, 2022 from 14:00-16:00 CEST.
Please feel free to add topics, contributions, or questions to the agenda at 
https://docs.google.com/presentation/d/1PbWzatrLbVOtc6m6UCBD97w8HNKJYWfQ_JeCga8Cod8/edit?usp=sharing

The VC information is:
https://uni-paderborn-de.zoom.us/j/95200773444?pwd=K1RCVzUyTjhIMk9VcnhTNTh3VENPZz09

Meeting ID: 952 0077 3444
Passcode: 104154

**Comment by oschuett** [comment]
Sorry to bother you. Something important came up and unfortunately I can't make it on the the 6th :-(
Maybe we could move it to the 7th? Otherwise, I'll simply add my comments to the slide deck.

**Comment by robertschade** [comment]
Hi Ole, from the schedule survey with your change the 7th now looks better than the 6th. I would suggest to move it.

@cp2k/cp2k-developers Please let us know till Wednesday evening if a move from the 6th to the 7th 14:00-16:00 CEST would be a problem for you.

**Comment by hfp** [comment]
I am btw available both dates and plan to attend (although my poll was saying different).

**Comment by Pibemanden** [comment]
Sorry about not filling out the survey. For what it is worth the 7th would be a better option for me, but I plan to attend regardless of which day the meeting ends up on.

**Comment by robertschade** [comment]
Thanks for the responses. In light of the responses let's shift to Thursday.

**Comment by robertschade** [comment]
**@cp2k/cp2k-developers Please note that the CP2K GPU Meeting was shifted to Thursday 7th 14:00-16:00 CEST.**

Dear @cp2k/cp2k-developers,
we would like to invite you to the CP2K GPU Developers meeting on Juli 7th, 2022 from 14:00-16:00 CEST.
Please feel free to add topics, contributions, or questions to the agenda at 
https://docs.google.com/presentation/d/1PbWzatrLbVOtc6m6UCBD97w8HNKJYWfQ_JeCga8Cod8/edit?usp=sharing

The VC information is:
https://uni-paderborn-de.zoom.us/j/95200773444?pwd=K1RCVzUyTjhIMk9VcnhTNTh3VENPZz09

Meeting ID: 952 0077 3444
Passcode: 104154

**Comment by mattatlincoln** [comment]
Hi, talking to Filippo Spiga from NVidia yesterday. They'd be interested in joining the discussion... Whether this meeting or a follow-up.
Licencing is a thing.
________________________________

                       
                        
The University of Lincoln, UK, is rated among the UK's top 20 universities for student satisfaction in the Guardian University Guide 2022 and the Complete University Guide 2022 and in the top 30 UK universities overall in the WhatUni Student Choice Awards 2020. We hold a top five-star rating in the QS Stars ratings system of global universities and are placed among the world's top 150 young universities in The Times Higher Education Young University Rankings 2021.
                         
The information in this e-mail and any attachments may be confidential. If you have received this email in error please notify the sender immediately and remove it from your system. Do not disclose the contents to another person or take copies.
​
Email is not secure and may contain viruses. The University of Lincoln makes every effort to ensure email is sent without viruses, but cannot guarantee this and recommends recipients take appropriate precautions.
​
The University may monitor email traffic data and content in accordance with its policies and English law. Further information can be found at: http://www.lincoln.ac.uk/legal. 
From: Robert Schade ***@***.***>
Sent: Sunday, June 5, 2022 4:50:52 PM
To: cp2k/cp2k-developers ***@***.***>
Cc: Matthew Watkins ***@***.***>; Team mention ***@***.***>
Subject: [cp2k/cp2k-developers] CP2K GPU Meeting (#26)


Dear @cp2k/cp2k-developers<https://github.com/orgs/cp2k/teams/cp2k-developers>,
you might remember that during the last CP2K developers meeting we had decided to have a meeting dedicated to CP2K and GPUs because the discussion would have been too special for the larger crowd of people that attended the CP2K developers meeting.
Thus, to find a good time slot for this CP2K-GPU meeting (2 hours) I would be happy if everyone who is invested or interested in this discussion could fill out the survey at
https://terminplaner.dfn.de/dDlBZkgrYurzxyaU
till the 15th of June.
Also, if you already have an aspect or topic in mind to talk about, please feel free to add it to the agenda and/or slides at
https://docs.google.com/presentation/d/1PbWzatrLbVOtc6m6UCBD97w8HNKJYWfQ_JeCga8Cod8/edit?usp=sharing
Many Thanks

**Comment by robertschade** [comment]
Hi @mattatlincoln, 
the meeting is unfortunately in the past. You can find the slides with notes at [https://www.cp2k.org/dev:meetings](https://www.cp2k.org/dev:meetings). We have regular CP2K developers meetings, and in the next meeting we could have a section on GPU support again.

**Comment by mattatlincoln** [comment]
Apologies, i missed this with the start of the teaching year. Thanks for the message.
________________________________

                       
                        
The University of Lincoln, UK, is rated among the UK's top 20 universities for student satisfaction in the Guardian University Guide 2022 and the Complete University Guide 2022 and in the top 30 UK universities overall in the WhatUni Student Choice Awards 2020. We hold a top five-star rating in the QS Stars ratings system of global universities and are placed among the world's top 150 young universities in The Times Higher Education Young University Rankings 2021.
                         
The information in this e-mail and any attachments may be confidential. If you have received this email in error please notify the sender immediately and remove it from your system. Do not disclose the contents to another person or take copies.
​
Email is not secure and may contain viruses. The University of Lincoln makes every effort to ensure email is sent without viruses, but cannot guarantee this and recommends recipients take appropriate precautions.
​
The University may monitor email traffic data and content in accordance with its policies and English law. Further information can be found at: http://www.lincoln.ac.uk/legal.
